ansible/templates/common/install.sh.j2

```diff
@@ -1,11 +1,17 @@
 {% if doc %}
-Install OMERO.web::
+Upgrade pip and install OMERO.web:
+
+.. parsed-literal::
+
+    /opt/omero/web//venv3/bin/pip install --upgrade pip
+    /opt/omero/web//venv3/bin/pip install "omero-web>=\ |version_web|"
```

Comment:
```suggestion
    /opt/omero/web//venv3/bin/pip install omero-web
```

Reply:
This does not match what we have put in place in the doc for other packages e.g. dropbox

Reply:
I understood this block was meant to be identical to https://github.com/ome/omeroweb-install/pull/77/commits/bf0e96566c4d21f68ec15902a84bfedd0992db69 with a different venv prefix?

No objection to keeping it as it is but for me the same question as https://github.com/ome/omeroweb-install/pull/77#discussion_r764128654 applies. Where is the dropbox line you are referring to?

Reply:
See https://github.com/ome/omero-documentation/pull/2109
To be honest, dropping the version makes this easier i.e. no need for ``if else``

Reply:
I have pushed a commit dropping the version